boards/arm/ks_zion/doc/index.rst

```diff
@@ -0,0 +1,127 @@
+.. _ks_zion_board:
+
+
+Kernelspace Zion
+################
+
+.. contents::
+
+
+Overview
+========
+
+The Kernelspace Zion is a small generic development board, featuring an
```

Comment:
you have very conservative line wrapping in this file -- please set your editor to use 100-char line length as per our coding guidelines